### ISSUE TITLE
Hi,it’s really a good work. However when I train it with voc 2 clipart, the results look like some wrong (about 20 map in my experiment which is not as good as other daod methods) . Can you tell me the reason?

### ISSUE THREAD

**Issue by ga546** [CityU-AIM-Group/SIGMA#22]

**Comment by wymanCV** [comment]
Hi, that is a good question. 

We found that the default hyper-parameters and training schedules in our EPM baseline (fcos-based detector) are not feasible for Pascal 2 Other settings. I have spent lots of time tuning these adaptation settings. After conducting the following modifications, the results will be comparable with SOTA.  

1) following TIA using both Pascal and style-transferred Pascal as the training set.
2) setting a larger value for GRL_WEIGHT_P{3-7}, e.g., 0.2
3) pre-training the model only with source-domain data after convergence.
4) reducing the learning rate to 0.0001 (it makes a big difference for fcos on Pascal VOC with inappropriate learning rate)

We plan to release the reproducible configs for these extra benchmark settings in our extended version. If you need the pre-trained model and configs in advance, you can leave an e-mail for me.  Thanks.

**Comment by ga546** [comment]
> 

I really need the pretrained model and configs about pascal 2 other settings and my qq email is 2951240141@qq.com. thank you very much.

**Comment by wymanCV** [comment]
> 2951240141@qq.com

I have sent you through e-mail. Thanks for your attention to our work.

**Comment by ga546** [comment]
> > [2951240141@qq.com](mailto:2951240141@qq.com)
> 
> I have sent you through e-mail. Thanks for your attention to our work.

thank you very much!
